### PR TITLE
Login and set password page

### PR DESCRIPTION
**Pull request by preranaprabhu-betsol**
Please review.

### CODE REVIEW

File: src/screens/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        },
+      };
+      const {
+        data: { success, error, data },
+      } = await axios.post("/api/auth/login", { email, password }, config);
+      if (success) {
+        setUserInfo(data);
+        localStorage.setItem("userInfo", JSON.stringify(data));
+      } else {
+        //to do snackbar in chakra
```
Comment (sriramgovindaraj) [inline src/screens/Login/Login.jsx]: Please add the snackbar here

File: src/screens/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        //message will be from backend (error )
+      }
+      setLoading(false);
+    } catch (error) {
+      //to do snackbar in chakra
```
Comment (sriramgovindaraj) [inline src/screens/Login/Login.jsx]: Here too

File: src/screens/PreOnboarding/PreOnboarding.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from "react";
```
Comment (sriramgovindaraj) [inline src/screens/PreOnboarding/PreOnboarding.jsx]: Please remove this file from this branch, you can add it in the other branch while building pre-onboarding

File: src/screens/Set_Password/set_password.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import { Formik, Field } from "formik";
```
Comment (sriramgovindaraj) [inline src/screens/Set_Password/set_password.jsx]: Remove this file too, copy it elsewhere, we can create a new branch for this feature too

**Review by sriramgovindaraj — CHANGES_REQUESTED** [review]

File: src/screens/Set_Password/set_password.jsx (hunk 4 above)
Comment (preranaprabhu-betsol) [reply src/screens/Set_Password/set_password.jsx]: okayyy

File: src/screens/PreOnboarding/PreOnboarding.jsx (hunk 3 above)
Comment (preranaprabhu-betsol) [reply src/screens/PreOnboarding/PreOnboarding.jsx]: yesss

**Review by sriramgovindaraj — APPROVED** [review]

**Pull request merged**